src/Transaction.php

```diff
@@ -284,4 +291,19 @@ public function setCallbackUrl( $callbackUrl )
 			return $this;
 		}
 
-	}
\ No newline at end of file
+
+				/**
+		 * Sets the metadata
+		 *
+		 * @param array $metadata
+		 *
+		 * @return $this
+		 */
+		public function setMetadata( array $metadata )
```

Comment:
Nice one adding type to the argument here. But I think it might cause issue for PHP users on version 5.6. Could we remove it? 🙏🏽

```php
function setMetadata( $metadata ) 
{ 
    //... 
}
```